api/node/src/types/brush.rs

```diff
@@ -1,23 +1,39 @@
 // Copyright © SixtyFPS GmbH <info@slint.dev>
 // SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-Royalty-free-1.1 OR LicenseRef-Slint-commercial
 
-use i_slint_core::{Brush, Color};
-use napi::bindgen_prelude::External;
+use i_slint_core::{graphics::GradientStop, Brush, Color};
+use napi::{bindgen_prelude::External, Error, Result};
 
 /// Color represents a color in the Slint run-time, represented using 8-bit channels for red, green, blue and the alpha (opacity).
-#[napi(js_name = Color)]
+#[napi(object, js_name = "Color")]
 pub struct JsColor {
+    /// Represents red channel of the color as u8 in the range 0..255.
```

Comment:
"Represents red channel" -> "Represents the red channel" (here and below for the other four)

Reply:
done